src/main/java/pl/wojciechkostecki/revel/model/MenuItem.java

```diff
@@ -0,0 +1,29 @@
+package pl.wojciechkostecki.revel.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+
+import javax.persistence.*;
+
+@Entity
+@Data
+public class MenuItem {
```

Comment:
And no Table annotation here, lol. Please, stay consistent and make all of the entity classes similar.